docs/source/yosys_internals/extending_yosys/build_verific.rst

```diff
@@ -93,6 +92,10 @@ Yosys builds:
 Partially supported builds
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. todo:: still unclear on the purpose of this section.
+
+   Are these the configurations we have tested as being able to compile?
```

Comment:
Yes, exactly. We know they compile, we think they will work correctly, but we don't work with those configurations ourselves so we don't know for sure, and if someone does run into an issue we can't guarantee we'll be able to fix it.